subprojects/testing-jvm/src/test/groovy/org/gradle/api/tasks/testing/TestTest2.groovy

```diff
@@ -43,7 +43,7 @@ import org.gradle.process.CommandLineArgumentProvider
 import static org.gradle.util.internal.WrapUtil.toLinkedSet
 import static org.gradle.util.internal.WrapUtil.toSet
 
-class TestTest extends AbstractConventionTaskTest {
+class TestTest2 extends AbstractConventionTaskTest {
```

Comment:
There was already a `TestTest` in this package in the `testing-jvm` project, which is where this class belongs.

Reply:
Can we combine these into a single test? The existing test in `testing-jvm` seems self-contained enough that it could just be copied into this file

Reply:
We can't, they extend different types.  I will rename `TestTest2` to `TestTaskConventionTaskTest` to make the distinction more clear.

Reply:
Damn, I didn't go deep enough.  OK, I'll merge them then.